src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+* Create a new alert condition 
+
+This doc will walk you through how to edit and change [an _existing_ alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#edit-existing-alert-condition), and creating [new_ alert condition_](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions/#create-alert-condition]). 
+
+## Create a new alert condition [#create-alert-condition]
+
+For this example, we are going to show you how to manually create a new alert condition using the **Alert Condition** UI. But, if you're just beginning to use New Relic, we suggest creating an alert condition from a chart. To get started with alerts, review our [tutorial about creating your first alert](/docs/tutorial-create-alerts/create-an-alert/). 
+
+<Steps>
+<Step>
+An alert condition is essentially a container that you create to define what conditions need to be met before you're notified of any unusual behavior.
+
+<img
+    title="set a query"
+    alt="A screenshot showing a user how to set a query for an alert condition"
+    src={alertsSetaQuery}
+/>
+
+<figcaption>
+To create a new alert, go to [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > **Alerts & AI**, select **Alert Conditions** in the left navigation, then select **New alert condition**.
+</figcaption>
+
+<CollapserGroup>
+  <Collapser
+    id="set-your-query"
+    title="Set your query"
+  >
+
+For this example, we will be using this query:
+
+```
+SELECT average(duration) 
+FROM PageView 
+WHERE appName = 'WebPortal'
+```
+Using this query for your alert condition tells New Relic that you want to know the average pageviews from your `WebPortal` application. Monitoring `pageviews` can help you look out for any latency issues in your application. 
+
+You can learn more about how to use NRQL, New Relic's query language, see our [NRQL documentation](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/). 
+
+</Collapser>
+</CollapserGroup>
+</Step>
+
+<Step>
+## Fine-tune advanced signal settings [#advanced-signal-settings]
+
+New Relic constantly observes the data that streams from your applications and into our system. But not all applications send signals at the same frequency or cadence. Some events could send signals to our system once a minute while others could only report data to New Relic once every day. An alert condition is a **specific** container designed for a **specific** use case. When creating an alert condition this section is the most customizable for the data that you're evaluating. 
+
+We're going to customize these advanced signal settings for our condition that is montioring `pageviews` in the `WebPortal` application. 
+
+<img
+    title="fine tune alert settings"
+    alt="A screenshot depicting advanced signal settings"
+    src={alertsFineTuneAlertSignals}
+/>
+
+<CollapserGroup>
```

Comment:
Before you jump into these collapsers, i'd add the next "scene" for your story. 

Scene 1: a user is concerned with page views, hence the initial query. 
Scene 2, fine tune your signal: What decision and subsequent action does a user name to make/do for this page to be valuable?

Reply:
i think laying out the example of how this page might be used will add value to the different variables they can modify, as described in your collapsers